pkg/madmin/hardware-commands.go

```diff
@@ -76,3 +79,44 @@ func (adm *AdminClient) ServerCPUHardwareInfo() ([]ServerCPUHardwareInfo, error)
 	}
 	return cpuInfo, nil
 }
+
+// ServerNetworkHardwareInfo holds informantion about cpu hardware
+type ServerNetworkHardwareInfo struct {
+	Addr        string          `json:"addr"`
+	Error       string          `json:"error,omitempty"`
+	NetworkInfo []net.Interface `json:"network"`
```

Comment:
We dont need a slice for the NetworkInfo

Reply:
This will be a slice as the network info needs to be appended for all the peers.